### PR TITLE
depend on tensorflow not tensorflow-gpu, drop holding h5py

### PR DESCRIPTION
**Pull request by bertsky**
- `tensorflow-gpu` seems to end at 2.4 line, while `tensorflow` is currently at 2.6
- h5py v2 was only necessary for Keras models in TF1 IIRC

### CODE REVIEW

**Comment by bertsky** [comment]
I have just tested: it still works with `tensorflow==2.6.3`

**Comment by cneud** [comment]
Running my test set...

**Review by cneud — APPROVED** [review]
